src/AzureAppConfigurationImpl.ts

```diff
@@ -532,6 +537,83 @@ export class AzureAppConfigurationImpl implements AzureAppConfiguration {
         }
         return response;
     }
+
+    async #parseFeatureFlag(setting: ConfigurationSetting<string>): Promise<any> {

```

Comment:
why async, same for line 561

Reply:
Line 597

https://stackoverflow.com/questions/57626477/using-javascript-crypto-subtle-in-synchronous-function